rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RMNodeStarter.java

```diff
@@ -779,6 +831,15 @@ protected String fillParameters(final CommandLine cl, final Options options) {
                 RMNodeStarter.ADD_NODE_ATTEMPTS_DELAY_IN_MS_USER_SUPPLIED = true;
             }
 
+            if (cl.hasOption(OPTION_AVAILABILITY_REPORT_TIMEOUT)) {
+                NODE_AVAILABILITY_REPORT_TIMEOUT_DELAY =
+                        Integer.valueOf(cl.getOptionValue(OPTION_AVAILABILITY_REPORT_TIMEOUT));
```

Comment:
It would be better to add a JVM property for this configuration, maybe using a property "instead" of a command line argument. Infrastructures uses the RMNodeStarter directly and do not allow extra command line arguments, where they allow extra JVM properties.